theano/sparse/tests/test_basic.py

```diff
@@ -2160,32 +2177,43 @@ def test_GetItem2D(self):
             assert r7.shape == t7.shape
             assert numpy.all(r7.toarray() == t7.toarray())
             """
+            if is_supported_version:
+                f4 = theano.function([x, a, b, e], x[a:b:e])
+                r4 = f4(vx, m, n, j)
+                t4 = vx[m:n:j]
+                assert r4.shape == t4.shape
+                assert numpy.all(t4.toarray() == r4.toarray())
+            else:
+                f4 = theano.function([x, a, b], x[a:b])
+                r4 = f4(vx, m, n)
+                t4 = vx[m:n]
+                assert r4.shape == t4.shape
+                assert numpy.all(t4.toarray() == r4.toarray())
```

Comment:
Remove duplicate assert, just keep one after the ifelse at the same level as the if.